internal/resources/fabric/service_token/resource_test.go

```diff
@@ -110,6 +110,53 @@ func TestAccFabricAsidePortServiceToken_PNFV(t *testing.T) {
 	})
 }
 
+func TestAccFabricZsidePortServiceToken_PNFV(t *testing.T) {
```

Comment:
You're adding tests for Zside, but then have documentation related to Aside. I'm confused how this is working.

Reply:
I added the wrong name in the documentation. It's zside service token example. Apologize for the confusion.